plugins/lookup/kustomize.py

```diff
@@ -55,6 +61,10 @@
 - name: Create kubernetes resources for lookup output with `--enable-helm` set
   kubernetes.core.k8s:
     definition: "{{ lookup('kubernetes.core.kustomize', dir='/path/to/kustomization', enable_helm=True) }}"
+
+- name: Create kubernetes resources for lookup output with environment variables
```

Comment:
Could you please add an example with a multiple-key dictionary? The attribute is supposed to be of type `dict`, but here you are defining it like `key=value`

Reply:
done